### ISSUE TITLE
support "ctrl-alt-<something>" keybindings

### ISSUE THREAD

**Issue by antoine-gallix** [junegunn/fzf#906]
- Category
    - [ ] fzf binary
    - [ ] fzf-tmux script
    - [x] Key bindings
    - [ ] Completion
    - [ ] Vim
    - [ ] Neovim
    - [ ] Etc.
- OS
    - [ ] Linux
    - [ ] Mac OS X
    - [ ] Windows
    - [ ] Windows Subsystem for Linux
    - [ ] Etc.
- Shell
    - [ ] bash
    - [ ] zsh
    - [ ] fish

Would it be possible to allow multiple-modifier keybindings? For example "ctrl-alt-h". For now, only keybindings with one modifier are supported.

**Comment by junegunn** [comment]
As far as I know, standard terminal emulators do not support multi-modifier key chords, so no, I don't think it's possible for the same reason you can't use them in other terminal programs (e.g. Vim). Correct me if I'm wrong.

**Comment by antoine-gallix** [comment]
Hi Junegunn, thanks for your quick answer.
I currently have the following lines in my in my `~/.inputrc` file on Linux:
```
"\C-j":backward-char
"\M-j":backward-word 
"\C-\M-j":beginning-of-line
```
`readline` is then able to differentiate between `Ctrl+x` `Alt+x` and `Ctrl+Alt+x` in every CL application using `readline`. I am not deeply aware of the different layers of terminal emulator, tty line policy, readline, shell that can each of them intercept and interpret keybindings, but if I can use `Ctrl+Alt+x` keybindings in a python shell inside fish inside my terminal emulator, I can believe it's also doable in FZF. But it's just a humble guess.

**Comment by junegunn** [comment]
Interesting, I'll look into it when I get some time. Thanks.

- junegunn

**Comment by junegunn** [comment]
Turns out that `ctrl-alt-x` is simply `ctrl-x` preceded by an escape character. I'll add support for the keys in the next version.

**Comment by junegunn** [comment]
Fixed in 0.16.7.

**Comment by antoine-gallix** [comment]
Wow, that was the quickest feature add I've ever seen. Impressive. Thanks @junegunn , I'm loving FZF even more !
